### PR TITLE
Feature/school 425/users logging

### PR DESCRIPTION
**Pull request by hecronk**

### CODE REVIEW

File: open_schools_platform/middleware/LastLoginIP.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from django.utils.deprecation import MiddlewareMixin
+
+from open_schools_platform.user_management.users.models import User
+
+
+class LastLoginIPMiddleware(MiddlewareMixin):
+    def __call__(self, request):
+        response = self.get_response(request)
+        if request.user.is_authenticated:
+            user = User.objects.get(id=request.user.id)
```
Comment (zavarin-michael) [inline open_schools_platform/middleware/LastLoginIP.py:10]: use get_user selector

File: open_schools_platform/user_management/users/serializers.py [hunk 2]
```diff
@@ -60,8 +59,15 @@ def create(self, validated_data):
         pass
 
 
-class UserSerializer(serializers.ModelSerializer):
+class HistoryRecordsField(serializers.ListField):
+    child = serializers.DictField()
```
Comment (zavarin-michael) [inline open_schools_platform/user_management/users/serializers.py:63]: what is it?

File: setup.cfg [hunk 3]
```diff
@@ -68,3 +68,7 @@ ignore_missing_imports = True
 [mypy-leaflet.*]
 # Remove this when django-phonenumber-field stubs are present
 ignore_missing_imports = True
+
+[mypy-simple_history.*]
+# Remove this when simple_history stubs are present
+ignore_missing_imports = True
```
Comment (zavarin-michael) [inline setup.cfg:74]: add new line in this file

**Review by zavarin-michael — CHANGES_REQUESTED** [review]

File: open_schools_platform/user_management/users/serializers.py (hunk 2 above)
Comment (hecronk) [reply open_schools_platform/user_management/users/serializers.py:63]: After calling the method .values() we need to check that data is correct

**Review by zavarin-michael — APPROVED** [review]

**Pull request merged**